copier/main.py

```diff
@@ -426,8 +426,21 @@ def _render_file(self, src_abspath: Path) -> None:
         if dst_relpath is None:
             return
         if src_abspath.name.endswith(self.template.templates_suffix):
-            tpl = self.jinja_env.get_template(str(src_relpath))
-            new_content = tpl.render(**self._render_context()).encode()
+            try:
+                tpl = self.jinja_env.get_template(src_relpath)
+            except UnicodeDecodeError:
+                if self.template.templates_suffix:
+                    # suffix is not emtpy, re-raise
+                    raise
+                # suffix is empty, fallback to copy
+                print(
+                    colors.warn
+                    | f"{src_relpath} could not be read as a Jinja2 template, falling back to a simple copy",
+                    file=sys.stderr,
+                )
```

Comment:
I prefer to silent this.

A warning printed to the user might make him think he did something wrong, while the template was working as expected.

It's better to add such warning in the template creation docs. Maybe under the docs for templates_suffix.

Reply:
Done!